web/backend/model.conf

```diff
@@ -0,0 +1,11 @@
+[request_definition]
+r = sub, obj, act
+
+[policy_definition]
+p = sub, obj, act
+
+[policy_effect]
+e = some(where (p.eft == allow))
+
+[matchers]
+m = r.sub == p.sub && r.obj == p.obj && r.act == p.act
```

Comment:
what this file does is not immediately obvious, can you add some documentation (ideally, inline) to indicate what each line does ?

Reply:
I added some quick comments.